buildpacks/java/java/func_yaml.go

```diff
@@ -34,14 +35,33 @@ func ParseFuncYaml(filedir string, logger bard.Logger) FuncYaml {
 
 	options := optionsToMap(cfg.Options, logger)
 	envs := envsToMap(cfg.Envs, logger)
-
+	name := getName(cfg.Envs, cfg.Name)
```

Comment:
Would `cfg.Name` be cleaner to get the name instead of iterating through `cfg.Envs`? https://pkg.go.dev/knative.dev/kn-plugin-func#Function

Reply:
`cfg` pulls from `func.yaml` though, while that env variable is a Python thing